lib/action_view/component/preview.rb

```diff
@@ -6,10 +6,9 @@ module ActionView
   module Component # :nodoc:
     class Preview
       extend ActiveSupport::DescendantsTracker
-      include ActionView::Component::TestHelpers
 
-      def render(component, *locals, &block)
-        render_inline(component, *locals, &block)
+      def render(component, **args, &block)
+        { component: component, args: args, block: block }
```

Comment:
don't do the render_inline thing from test helpers. Instead just minim the api and return the args. We'll later pass them to a true `render` method.